### DISCUSSION TITLE
Unable to get og metadata on youtube

### DISCUSSION THREAD

**Discussion by masashai** [sparklemotion/mechanize#654] [Q&A]
I can get it from localhost, but not from the production environment. The production environment is using Heroku.

using version
`mechanize (2.8.1)`

**Comment by flavorjones** [comment]
@masashai Sorry you're having a problem. But without more information I am not going to be able to help you.

Can you please provide:
- a self-contained ruby script that reproduces what you're seeing
- the error message that you're seeing in the failing environment

**Reply by masashai** [reply]
Sorry, solved. 

There was no problem with the mechanize.

It seemed that depending on the request header, the og metadata on youtube was not being output. I still don't know about the difference between localhost and the production environment. However, there is most likely a problem with YouTube.

Thanks.

Reference URL:
https://stackoverflow.com/questions/46616703/finding-open-graph-tags-on-youtube
